example/transform/openai_pdf_source_10k_summary.ipynb

```diff
@@ -200,32 +290,34 @@
   },
   {
    "cell_type": "markdown",
+   "id": "8cf645d3",
```

Comment:
shall we add `\` inside the `"""` statement to remove the initial space on each line?
[https://google.github.io/styleguide/pyguide.html](url)